packages/babel-cli/test/index.js

```diff
@@ -244,7 +246,7 @@ fs.readdirSync(fixtureLoc).forEach(function (binName) {
 
           delete taskOpts.os;
         }
-        merge(opts, taskOpts);
+        opts = { args: [], ...taskOpts };
```

Comment:
Changed this merge too (the only thing replaced is args, which is always set in our tests given it's babel-cli)